homeassistant/components/sensor/__init__.py

```diff
@@ -210,7 +210,7 @@ class SensorDeviceClass(StrEnum):
     ENERGY = "energy"
     """Energy.
 
-    Unit of measurement: `Wh`, `kWh`, `MWh`, `GJ`
+    Unit of measurement: `Wh`, `kWh`, `MWh`, `GJ`, `GWh`, `TWh`
```

Comment:
> GWh, TWh are also energy units

Sure, but that doesn't make them supported. Adding a comment doesn't fix/resolve it.
Additionally, we only add these when there is a use case for them raised as an architectural discussion.